CiE-iOS/CiE-iOS/ViewControllers/SettingsViewController.swift

```diff
@@ -10,6 +10,10 @@ import UIKit
 
 class SettingsViewController: UIViewController {
     
+    @IBAction func LogoutButtonPressed(_ sender: Any) {
```

Comment:
Please keep in mind (focus on ) what exactly you issue is do not implement other functionality just because it is convenient.
This button should be part of the Settings View Controller issue.

Please do remove this button.